vertx-web/src/main/java/io/vertx/ext/web/handler/sockjs/impl/EventBusBridgeImpl.java

```diff
@@ -590,8 +592,12 @@ private boolean regexMatches(String matchRegex, String address) {
     return m.matches();
   }
 
-  private static void replyError(SockJSSocket sock, String err) {
-    JsonObject envelope = new JsonObject().put("type", "err").put("body", err);
+  private static void replyError(SockJSSocket sock, String type, String message) {
+    JsonObject envelope = new JsonObject()
```

Comment:
I think we need to keep the old behavior as well to avoid breaking change and test it.

Reply:
@vietj Do you mean using 'body' key for error messages?

Reply:
yes

Reply:
@vietj 

IMHO the existing behavior should be changed you can check the Vert.x TCP EventBus bridge uses “message” key instead of “body” to report descriptive message for error type messages.

https://github.com/vert-x3/vertx-tcp-eventbus-bridge/blob/master/src/main/java/io/vertx/ext/eventbus/bridge/tcp/impl/protocol/FrameHelper.java#L86